### PR TITLE
CLDR-14163 Add missing character to parseLenients

### PR DESCRIPTION
**Pull request by sffc**
This should fix the failing ICU exhaustive test.

I only updated these sets in the root.  I did not touch any that might exist in child locales.

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14163
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by pedberg-icu** [comment]
Note: Adding
1C79;OL CHIKI GAAHLAA TTUDDAAG to period equivalents
1C7C;OL CHIKI PHAARKAA to hyphen/dash equivalents
2E41;REVERSED COMMA to comma equivalents
seems reasonable

**Review by pedberg-icu — DISMISSED** [review]

File: common/main/root.xml
Comment (macchiati) [inline common/main/root.xml:75]: This REVERSED COMMA looks ok (unlikely to ever happen) but it is a punctuation character at least.

Comment (macchiati) [reply common/main/root.xml:75]: So we could take this but not the others, and make 2 changes in ICU. 

Or just make 3 changes in ICU.

**Review by macchiati — DISMISSED** [review]

File: common/main/root.xml
Comment (sffc) [reply common/main/root.xml:70]: It is used in locale "sat" as the minus sign, which is why I added it here.  parseLenients covers all minus signs in all locales.  (If that should no longer be the case, and ICU change will be required)

https://github.com/unicode-org/cldr/blob/1d75dd4595e30f34b45c240112087e0d2105d2f7/common/main/sat.xml#L667

File: common/main/root.xml
Comment (sffc) [reply common/main/root.xml:59]: Similarly, this character is used as the decimal separator in locale "sat".

File: common/main/root.xml
Comment (sffc) [reply common/main/root.xml:75]: I don't understand what you mean by "just make 3 changes in ICU".  The failing ICU test is a data-driven test that ensures that parseLenients is a superset of the number formatting symbols in all locales.

**Comment by macchiati** [comment]
So I recommend the following changes to this PR.

1. Change sat.xml to fix the following two lines to use ASCII
663: <decimal>ᱹ</decimal> 
667: <minusSign>ᱼ</minusSign> 

2. Drop the changes to the lenient characters adding 
1C79;OL CHIKI GAAHLAA TTUDDAAG to period equivalents
1C7C;OL CHIKI PHAARKAA to hyphen/dash equivalents

And file a ticket to add a check to decimal and grouping that they are in the <parseLenients scope="number" > chararacters, and otherwise cause an error to users (asking them to flag the item and leave a forum post justifying the usage).

**Comment by pedberg-icu** [comment]
We should be careful about using ASCII period. Here is what the Unicode book says about Ol Chiki punctuation:
Punctuation. Western-style punctuation, such as the comma, exclamation mark, question mark, and quotation marks are used in Ol Chiki text. U+002E “.” full stop is not used, because it is visually confusable with the modifier letter U+1C79 ol chiki gaahlaa ttud- daag.

**Comment by macchiati** [comment]
I agree, for sentence punctuation. This would be restricted to numbers.

Mark

**Comment by pedberg-icu** [comment]
I am waiting on this to start CLDR-ICU integration. Is this likely to be addressed soon? If not I can close this and start a PR of my own to do this.

**Comment by sffc** [comment]
I updated the PR.  The ticket still needs to be accepted before we merge.

**Comment by pedberg-icu** [comment]
> I updated the PR. The ticket still needs to be accepted before we merge.

Thanks for the chnages. I sent an e-mail to cldr-core2 about setting ticket to Accepted, will do that shortly. In the meantime can you go ahead and squash? Then I will re-review. Thanks.

**Review by pedberg-icu — APPROVED** [review]

**Comment by sffc** [comment]
Squashed.  Feel free to merge.

**Comment by pedberg-icu** [comment]
> Squashed. Feel free to merge.

Thanks!! I will wait for tests to finish then merge.

**Review by pedberg-icu — APPROVED** [review]

**Review by macchiati — COMMENTED** [review]
LGTM

**Pull request merged**
